.packit.yaml

```diff
@@ -32,19 +32,40 @@ jobs:
     metadata:
       dist_git_branches:
         - fedora-all
+  - job: propose_downstream
+    trigger: release
+    files_to_sync:
+      # There is a different spec-file for epel-8:
+      - fedora/python-specfile-epel-8.spec
```

Comment:
I was thinking about something like `epel/python-specfile.spec`, but that's only a detail.

Reply:
But maybe it would help with the Copr issue?

Reply:
OK, I'll try that.